src/applications/edu-benefits/1990/containers/ConfirmationPage.jsx

```diff
@@ -161,11 +165,20 @@ class ConfirmationPage extends React.Component {
             What happens after I apply?
           </h4>
           <p className="confirmation-guidance-message">
+            We usually decide on applications within 30 days.
+            <br />
+            <br />
+            You’ll get a Certificate of Eligibility (COE) or decision letter in
+            the mail. If we’ve approved your application, you can bring the COE
+            to the VA certifying official at your school.
+            <br />
+            <br />
```

Comment:
Is there a better way to style the whitespace here?

Reply:
I can modify that a bit. The double `<br />` is certainly not ideal.